packages/kolibri-tools/lib/i18n/crowdin/utils.py

```diff
@@ -146,9 +146,9 @@ def json_dump_formatted(data, file_path, file_name):
             )
 
 
-def read_config_file():
+def read_config_file(path=None):
     output = {}
-    config_file = os.path.join(os.getcwd(), "setup.cfg")
+    config_file = os.path.join((path or os.getcwd()), "setup.cfg")
```

Comment:
As I noted above, I don't think this change is necessary, and is likely to make the behaviour unstable.